sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -173,7 +175,7 @@ class EthService(
 
   private def doCall(nodeView: NV, params: TransactionArgs, tag: String): Array[Byte] = {
     getStateViewAtTag(nodeView, tag) { (tagStateView, blockContext) =>
-      val msg = params.toMessage(blockContext.baseFee)
+      val msg = params.toMessage(blockContext.baseFee, BigInteger.valueOf(settings.globalRpcGasCap))
```

Comment:
I think we discussed this before, but I'm not sure anymore:
I think we can use `Long`, i.e. signed 64 bit integer everywhere for gas, instead of `BigInteger`. GETH uses unsigned 64 bit integer, but even with 63 available bits the numbers are already way above anything that will ever be used.

Reply:
let's discuss it during the standup. There was a parallel thread with Alberto and Sara on the similar topic. As an outcome we decided to stay in a safe side with BigInteger.

Reply:
let's use BigInteger

Reply:
done